tools/pfe-tools/11ty/DocsPage.ts

```diff
@@ -11,7 +11,7 @@ import nunjucks, { Environment } from 'nunjucks';
 
 interface DocsPageOptions extends PfeConfig {
   docsTemplatePath?: string;
-  tagName: string;
```

Comment:
Was looking for the reasoning for this change.  Was there just a typescript warning on this?  Was the nullish coalescing not enough later in usage?  Just curious if it was more cleanup then anything, noticed other options were all optional as well.

Reply:
can't specifically recall, but I believe my intention was to have the type more accurately reflect call-site usage